contracts/profiles/DNSResolver.sol

```diff
@@ -0,0 +1,142 @@
+pragma solidity ^0.5.0;
+
+import "../ResolverBase.sol";
+import "../util/RRUtils.sol";
```

Comment:
Could we import this from one of the existing NPM packages, instead of copying it?

Reply:
Would love to; do you have info on which NPM package I should be importing?

Reply:
I believe it should be in `@ensdomains/dnssec-oracle` at `@ensdomains/dnssec-oracle/contracts/RRUtils.sol`.